ddtrace/llmobs/_llmobs.py

```diff
@@ -322,7 +330,11 @@ def _llmobs_span_event(self, span: Span) -> LLMObsSpanEvent:
             "meta": meta,
             "metrics": metrics,
             "tags": [],
-            "_dd": {"span_id": str(span.span_id), "trace_id": format_trace_id(span.trace_id)},
+            "_dd": {
+                "span_id": str(span.span_id),
+                "trace_id": format_trace_id(span.trace_id),
+                "apm_trace_id": format_trace_id(span.trace_id),
```

Comment:
Why is the trace_id the same as apm_trace_id? Should it instead match the trace_id on line 323?
```suggestion
                "trace_id": format_trace_id(llmobs_trace_id),
                "apm_trace_id": format_trace_id(span.trace_id),
```

Reply:
`._dd.trace_id` and `._dd.span_id` were added a few months ago by @Yun-Kim i believe specifically to house the apm ids, on the chance that we wanted to make this migration. in effect, this pr is both establishing an llmobs trace_id and renaming the `._dd.trace_id` field to `._dd.apm_trace_id`, but leaving the former in on the chance that it is used for something somewhere.

Reply:
I get it now, fair enough, let me resolve this.